### PR TITLE
Remove shortstop handler from example

### PR DESCRIPTION
**Pull request by jasisk**
... as pointed out in #50 by @rapzo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ http.createServer(app).listen(8080);
         "priority": 30,
         "route": "/foo",
         "module": {
-            "name": "path:./lib/middleware",
+            "name": "./lib/middleware",
             "method": "customMiddleware",
```
Comment (pvenkatakrishnan) [inline README.md:52]: why?

Comment (jasisk) [inline README.md:52]: This is meddleware, not shortstop. :grinning:

Comment (pvenkatakrishnan) [inline README.md:52]: gah :+1:

**Comment by jasisk** [comment]
Readme change. Execumerging.

**Pull request merged**